assets/_pre_prod/publish/v1.0/features/de.txt

```diff
@@ -1,10 +1,11 @@
-Etar ist ein im Material Design gestalteter open source Kalender für jedermann!
-Besonderheiten:
-★ Monatsansicht
-★ Wochen, Tag und Agendaansicht.
-★ Benutzt Android Calendar sync. Arbeitet mit Google-Kalender, Exchange usw. zusammen.
-★ Material design.
+Etar ist ein im Material Design gehaltener Open Source-Kalender für jedermann!
```

Comment:
I would remove open source here since it is in line 9.
If you want to keep it, it should be "Open-Source-Kalender". See rule 26: http://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich